src/lib/net/address.c

```diff
@@ -818,7 +818,8 @@ tor_addr_is_loopback(const tor_addr_t *addr)
 /* Is addr valid?
  * Checks that addr is non-NULL and not tor_addr_is_null().
  * If for_listening is true, IPv4 addr 0.0.0.0 is allowed.
- * It means "bind to all addresses on the local machine". */
+ * It means "bind to all addresses on the local machine".
+ * Supports checking IPv6 adresses. */
```

Comment:
Thanks for this extra comment, but it's not really needed. All Tor's address functions support IPv4 and IPv6. (Unless we have named or documented them to only support particular kinds of addresses.)

Normally extra comments are ok. But someone else is changing this function in another ticket, and if you both change nearby lines, we will get merge conflicts. Here is the other ticket:
https://trac.torproject.org/projects/tor/ticket/33679#comment:3